hw2/hw2/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace дз2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double sphereSquare, sphereValue;
```

Comment:
Объем по-английски _volume_.
```suggestion
            double sphereSquare, sphereVolume;
```